sleap/gui/commands.py

```
@@ -49,6 +49,7 @@ class which inherits from `AppCommand` (or a more specialized class such as
 from sleap.gui.dialogs.merge import MergeDialog, ReplaceSkeletonTableDialog
 from sleap.gui.dialogs.message import MessageDialog
 from sleap.gui.dialogs.missingfiles import MissingFilesDialog
+from sleap.gui.dialogs.export_clip import ExportClipAndLabelsDialog
```

Ruff (0.8.2)

`sleap.gui.dialogs.export_clip.ExportClipAndLabelsDialog` imported but unused

Remove unused import: `sleap.gui.dialogs.export_clip.ExportClipAndLabelsDialog`

(F401)